src/test/java/io/openliberty/tools/intellij/it/MavenSingleModMPProjectTest.java

```diff
@@ -96,6 +111,16 @@ public String getWLPInstallPath() {
         return WLP_INSTALL_PATH;
     }
 
+    /**
+     * Returns the path of server.env file.
+     *
+     * @return The path of server.env file.
+     */
+    @Override
+    public String[] getServerEnvPath() {
+        return new String[]{SM_MP_PROJECT_NAME, TARGET_DIR, LIBERTY_DIR, "wlp", "usr", "servers", "defaultServer", "server.env"};
```

Comment:
When using `WLP_INSTALL_PATH` we are getting `Paths.get("target", "liberty").toString();` which gives us `target/liberty`. For the function to open the required file we need to get `target, liberty`. So using `WLP_INSTALL_PATH` was not possible. But in the case of Gradle there doesnt seem to be a problem because the value of `WLP_INSTALL_PATH` is `build`.

Reply:
This is also no longer necessary as we're only reading the server.env file instead of opening it.